packages/p2p/src/components/my-ads/__tests__/ad-status.spec.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import AdStatus from '../ad-status.jsx';
+
+describe('<Verification />', () => {
```

Comment:
should not this  change to` <AdStatus />` ?

Reply:
Sure, thx for notice!